test/src/components/Field-test.js

```diff
@@ -14,30 +14,6 @@ describe('Field', () => {
       .findText('yo child');
   });
 
-  it('can be small', () => {
-    render(<Field small />).findClass('small');
-  });
-
-  it('can be large', () => {
-    render(<Field large />).findClass('large');
-  });
-
-  it('can be inline', () => {
-    render(<Field inline />).findClass('inline');
-  });
-
-  it('can be disabled', () => {
-    render(<Field disabled />).findClass('disabled');
-  });
-
-  it('can be required', () => {
-    render(<Field required />).findClass('required');
-  });
-
-  it('can have an error', () => {
-    render(<Field error />).findClass('error');
-  });
-
```

Comment:
All these bool prop tests will be replaced with the single Stardust component guideline test.  It will ensure classes and props are inherited correctly.  We will run it against all components.  See #33.